packages/jest-enzyme/src/toMatchSnapshot.js

```diff
@@ -0,0 +1,21 @@
+import wrapperToJSON from 'enzyme-to-json';
+
+const GLOBAL_STATE = Symbol.for('$$jest-matchers-object');
+const originalToMatchSnapshot = global[GLOBAL_STATE].matchers.toMatchSnapshot;
```

Comment:
specifically, is this safe? Is there a better way to get access to this reference?